### ISSUE TITLE
BUG: A border on the mj-column is shown twice in the editor

### ISSUE THREAD

**Issue by adiux** [GrapesJS/mjml#290]
### What browser are you using?

Chrome 102.0.5005.61

### Reproducible demo link

https://codepen.io/adiux/pen/BaYrEXR?editors=1111

### Describe the bug

**How to reproduce the bug?**
1. Add grapesjs-mjml to the project with any version newer than 0.5.6
2. Add a border to a section. E.g.: `<mj-column border="4px solid #F45E43">`
3. Open MJML template with grapesjs

**What is the expected behavior?**
only one border is shown:
<img width="792" alt="image" src="https://user-images.githubusercontent.com/13075514/171250244-f96c395b-5a6e-4cde-a7f4-f60d778c9a15.png">

**What is the current behavior?**
two borders are shown:
<img width="766" alt="image" src="https://user-images.githubusercontent.com/13075514/171249926-6ed65f00-c09e-4c0c-a668-94fcc59fca0d.png">

If is necessary to execute some code in order to reproduce the bug, paste it here below:
```js
// Working
import grapesjsMjml from "https://cdn.skypack.dev/grapesjs-mjml@0.5.6";

// Not working
import grapesjsMjml from "https://cdn.skypack.dev/grapesjs-mjml@0.5.7";
import grapesjsMjml from "https://cdn.skypack.dev/grapesjs-mjml@0.6.0";
```

> Note: The rendered HTML is fine. It is only an issue with the view in the editor.

**Comment by DRoet** [comment]
PRs are welcome to fix this, I currently don't have time to look into this myself

**Comment by DRoet** [comment]
probably related to the changes in https://github.com/artf/grapesjs-mjml/pull/270 which was released in v0.5.7

**Comment by adiux** [comment]
Thanks for looking into this. For me it was not working with v0.6.0:
import grapesjsMjml from "https://cdn.skypack.dev/grapesjs-mjml@0.6.0";

**Comment by gustavohleal** [comment]
The thing is that grapesjs-mjml puts a div as the mj-column main element on the canvas. Inside it the column is rendered by the mjml engine. When the background-color issue was fixed in #270 by adding the styles to the div element, the border is duplicated and the white space between borders is the padding of the div.
I'm fixing this on the project i'm working on and later i'll copy to a raw clone of the grapesjs-mjml repository and then open a PR.
For those who need the fix asap, I just filtering the styles for the attributes that need to be in the div by putting the names inside `styleFix` array. For now this is working and keeping the solution from #270. If anyone has an ideia about how to get this better, please let me know.
```javascript
            renderStyle() {                
                const model_style = this.model.get('style') || {}
                const stylesFix = ["background-color"];
                const otherStyle = Object.keys(this.model.get('style')).map(attr=>`${attr}:${model_style[attr]};`);
                let style = [];  
                let styleKeys = Object.keys(this.model.get('style'));
                stylesFix.forEach(attr => {
                    if(styleKeys.includes(attr)){
                        style.push(`${attr}:${model_style[attr]};`)
                    }
                })                
                this.el.setAttribute('style', `${this.attributes.style} ${style.join(' ')} ${this.el.getAttribute('style')}`);
                
            },
```

**Comment by gustavohleal** [comment]
Even before #270 the border was added to the table inside the div. With the solution of adding the styles to the div the border is being added to both of them. This is whats causing this bug. I'm not finding where this is passed to the inner element. But when you change the padding of the column the element is rendered again and the problem is fixed. Something in the builtin property of border is causing this. Border-radius is also passed to the table and not rendered.
